packages/common/src/utils/formatting.ts

```diff
@@ -110,19 +110,21 @@ function roundNumber(
     const isValidFormat =
         !!format && format !== 'km' && format !== 'mi' && format !== 'percent';
 
+    const validFractionDigits = invalidRound
```

Comment:
This change just takes care of errors on providing negative rounds. But doesn't implement negative rounds as described in doc: 


Value:       121.854
round:       -2
expect:     100

I didn't understand the concept of negative rounding in doc though. What should be the expectation if the round is -1 on the above example ?

Reply:
Closed that ticket in favour of https://github.com/lightdash/lightdash/issues/1990